app/jobs/plans/update_subscription_fee_job.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+module Plans
+  class UpdateSubscriptionFeeJob < ApplicationJob
```

Comment:
Since `Fee` refers to another resource, what to do you thing of naming this job `UpdateAmoutJob`?